README.md

```diff
@@ -11,55 +15,48 @@
 [![Codecov](https://codecov.io/gh/me-ica/tedana/branch/master/graph/badge.svg)](https://codecov.io/gh/me-ica/tedana)
 [![Join the chat at https://gitter.im/ME-ICA/tedana](https://badges.gitter.im/ME-ICA/tedana.svg)](https://gitter.im/ME-ICA/tedana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-   ![](https://user-images.githubusercontent.com/7406227/40031156-57b7cbb8-57bc-11e8-8c51-5b29f2e86a48.png)
+About
+-----
 
+``tedana`` originally came about as a part of the [ME-ICA](https://github.com/me-ica/me-ica) pipeline.
+The ME-ICA pipeline originally performed both pre-processing and TE-dependent
+analysis of multi-echo fMRI data; however, ``tedana`` now assumes that you're
+working with data which has been previously preprocessed.
+If you're in need of a preprocessing pipeline, we recommend
+[fmriprep](https://github.com/poldracklab/fmriprep/), which has been tested
+for compatibility with multi-echo fMRI data and ``tedana``.
 
-## About
+![http://tedana.readthedocs.io/](https://user-images.githubusercontent.com/7406227/40031156-57b7cbb8-57bc-11e8-8c51-5b29f2e86a48.png)
+![](https://github.com/tsalo/tedana/blob/doc-pipeline/docs/_static/tedana-poster.png)
```

Comment:
Could we remove this from the README and put it on the landing page of the RTD site ? Maybe under citations ?